build-a-game-playing-agent/agent_test.py

```diff
@@ -95,18 +94,16 @@ def testWrapper(self):
                     raise err[0](err[1]).with_traceback(err[2])
                 return res
             except QueueEmptyError:
-                raise TimeoutError("Test aborted due to timeout. Test was " +
-                    "expected to finish in less than {} second(s).".format(time_limit))
+                raise TimeoutError("Test aborted due to timeout. Test was "
+                                   "expected to finish in less than {} second(s).".format(time_limit))
 
         return testWrapper
 
     return wrapUnitTest
 
 
 def makeEvalTable(table):
-
```

Comment:
Still not clear why the blank lines need to be removed.

See also, e.g., lines 109, 149, and 182, etc.